packages/jetpack-logo/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "automattic/jetpack-logo",
+	"description": "A logo for Jetpack",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"version": "1.0.0",
```

Comment:
I know we don't have real Composer packages now, but once we do, the `versions` should be omitted.